qdrant_docs_mcp/tools/importer.py

```diff
@@ -232,9 +324,7 @@ def import_snippets(
     for snippet in track(snippets, description="Uploading snippets..."):
         # Combine description and snippet for embedding
         embedding = next(
-            embedding_provider.embedding_model.embed(
-                [snippet.category + snippet.description]
-            )
+            embedding_provider.embedding_model.embed([snippet.description])
```

Comment:
let's add a check for an empty description

Reply:
resolved by rebase